annoyed-alligators/socl_media/apps/terminal/views.py

```diff
@@ -0,0 +1,34 @@
+from django.shortcuts import render
+from django.http import JsonResponse, HttpResponse
+
+import random
+import json
+
+from .methods import TerminalCommand
+# Create your views here.
+
+def index(request):
+    # names = ("bob", "dan", "jack", "lizzy", "susan")
+    # items = []
+    # for i in range(100):
+    #     items.append({
+    #         "name": random.choice(names),
+    #         "age": random.randint(20, 80),
+    #         "url": "https://example.com",
+    #     })
+
+    # context = {}
+    # context["items"] = json.dumps(items)
+    return render(request, 'terminal.html')
+
+
+def terminal_command(request):
```

Comment:
I think this can have a more expressive name, something like `terminal_command_execute` maybe?